marketplace/wpp_templates/tests/test_requests.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
+from unittest.mock import patch
+
+from marketplace.core.tests.base import FakeRequestsResponse
+
+from ..requests import TemplateMessageRequest
+from marketplace.core.types.channels.whatsapp_base.exceptions import FacebookApiException
+
+
+class TemplateMessageRequestTestCase(TestCase):
```

Comment:
there are only tests that validate an error in the request, create tests for when it returns 200